server/app/repository/AccountRepository.java

```diff
@@ -464,6 +465,21 @@ public Query<ApplicantModel> findApplicantsWithIncorrectDobPath() {
     return database.findNative(ApplicantModel.class, sql);
   }
 
+  /**
+   * For use in {@link MigratePrimaryApplicantInfoJob}. Will return all applicants that have data in
+   * well known paths.
+   */
+  public Query<ApplicantModel> findApplicantsNeedingPrimaryApplicantInfoDataMigration() {
```

Comment:
question (blocking): Do we need tests for this?

Reply:
+1 to tests